File: Modul 2/Soal 10/PRAK110-2210817310014-Fatih Alif Ghifari.py
import math
a=5
t=12
SB=math.sqrt((a*a)+(t*t))
k=a+t+SB
l=(a*t)/2
print("Diketahui :")
print("Alas = {} cm".format(a))
print("Tinggi = {} cm\n".format(t))
print("Jawab :")
print("Sisi A = {} cm".format(t))
print("Sisi B = {} cm".format(round(SB)))
print("Sisi C = {} cm".format(a))
print("Keliling = {} cm".format(round(k)))
print("Luas = {} cm".format(round(l)))

File: Modul 2/Soal 10/PRAK110-2210817310014-Fatih Alif Ghifari.c
#include <stdio.h>
#include <math.h>

int main()
{
    int a, b, SA, SB, SC, k, l, axt;
    a=5;
    b=12;
    SA=a;
    SC=b;
    axt=(a*a) + (b*b);
    SB=sqrt(axt);
    k=SA+SB+SC;
    l=(SA*SC)/2;
    printf("Diketahui : \n");
    printf("Alas = %d cm\n", a);
    printf("Tinggi = %d cm\n\n", b);
    printf("Jawab :\n");
    printf("Sisi A = %d cm\n", SA);
    printf("Sisi B = %d cm\n", SB);
    printf("Sisi C = %d cm\n", SC);
    printf("Keliling = %d cm\n", k);
    printf("Luas = %d cm\n"), l;
    return 0;
}